rasa/nlu/selectors/response_selector.py

```diff
@@ -635,17 +635,17 @@ def load(
     def _uses_sequence_features_for_text(self) -> bool:
```

Comment:
Shouldn't this be `_uses_sequence_features_for_input_text`?

Reply:
ouch, good catch